examples/index.html

```diff
@@ -177,6 +178,18 @@ <h4>Amazon KVS WebRTC DQP</h4>
                 </div>
             </div>
 
+            <h4>Amazon KVS WebRTC Datachannel Benchmarking</h4>
+            <div class="form-group">
+                <div class="form-check form-check-inline">
+                    <input class="form-check-input" type="checkbox" id="enableDatachannelBenchmarking" value="enableDatachannelBenchmarking">
+                    <label for="enableDatachannelBenchmarking" class="form-check-label">Enable KVS WebRTC Datachannel Benchmarking <small>(Master + Viewer)</small></label>
```

Comment:
The line suggests we are benchmarking data channel. Can we change to `Enable C SDK master profile timeline`? This is more accurate. Lets change the label too to something similar like `enableCMasterProfileTimeline`

Reply:
Renamed it to enableProfileTimeline as we are plotting master and viewer both